opm/output/eclipse/WriteInit.cpp

```diff
@@ -489,7 +489,7 @@ namespace {
         auto depth = singlePrecision(lgr_grid.getLGRCell_all_depth(grid));
 
         for (auto cell = 0*nAct; cell < nAct; ++cell) {
-            const auto  globCell = lgr_grid.getGlobalIndex(cell);
+            const auto globCell = grid.getLGR_father(cell, lgr_grid.get_lgr_tag());
```

Comment:
Function [`getLGR_father()`](https://github.com/OPM/opm-common/blob/cef88885329ed1babb1ab3184bcb95dbc02b3a0e/opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp#L321) **appears** to take a `global_index` as its (first) input parameter, but we're passing a value in the range `0 .. lgr_grid.getNumActive()-1` (i.e., `nAct-1`) here. If there are no inactive cells in the local grid, then the two are of course the same, but in general they will not be.

Do we perhaps need another layer of indirection to guarantee that we're looking up the expected host/parent cell?